server/cmd/code-annotation/handler/assignments.go

```diff
@@ -0,0 +1,74 @@
+package handler
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/render"
+
+	"github.com/src-d/code-annotation/server/cmd/code-annotation/serializer"
```

Comment:
do you think it's better to move it out of cmd? It's a common pattern in go to keep only entry point in cmd folder.
Please let me know if you need more information about it.

Reply:
Sure!!!
That aligns quite well with [Alex comment](https://github.com/src-d/code-annotation/pull/16#issuecomment-358920441)
I tried to follow the initial skeleton and I might misunderstand it.
I'd propose the following https://github.com/src-d/code-annotation/pull/16#issuecomment-358947367